doc/colvars-refman-main.tex

```diff
@@ -6489,6 +6489,383 @@
+    Frequency of hill creation}{%
+    positive integer}{%
+    The frequency for kernel deposition.}
+\item %
+  \labelkey{opes_metad|biasfactor}
+  \keydef
+    {biasfactor}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    The $\gamma$ biasfactor used for the well-tempered target distribution}{%
+    positive decimal}{%
+    $\mathrm{barrier} / k_{\mathrm{B}} T$}{%
+    This option sets the $\gamma$ in Eq.(\ref{eq:opes_potential}). It cannot be "inf" in the exploration mode.}
+\item %
+  \labelkey{opes_metad|sigma}
+  \key
+    {gaussianSigma}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    The initial widths of the kernels (divided by the square root of gamma in exploration mode)}{%
+    positive decimal}{%
+    This option sets the initial $\sigma$ values of the Gaussian kernels. The number of \texttt{gaussianSigma} values should match the number of CVs used by \texttt{colvars}. If \refkey{adaptiveSigma}{opes_metad|adaptive_sigma} is set, then this option can be ignored.}
+\item %
+  \labelkey{opes_metad|sigma_min}
+  \key
+    {gaussianSigmaMin}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    The minimum widths of the kernels (divided by the square root of gamma in exploration mode)}{%
+    positive decimal (optional)}{%
+    This option sets the minimum $\sigma$ values of the Gaussian kernels. The number of \texttt{gaussianSigmaMin} values should match the number of CVs used by \texttt{colvars}. The OPES method scales down the $\sigma$ during the simulation, and will never shrink $\sigma$ under \texttt{gaussianSigmaMin} if this option is set.}
+\item %
+  \labelkey{opes_metad|explore}
+  \keydef
+    {explore}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    Exploration mode}{%
+    boolean}{%
+    \texttt{off}}{%
+    If enabled, the OPES exploration mode will be used (see Ref.\cite{Invernizzi2022} for more information about the exploration mode).}
+\item %
+  \labelkey{opes_metad|epsilon}
+  \keydef
+    {epsilon}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    The value of the regularization constant for the probability}{%
+    positive decimal}{%
+    $\exp(-\mathrm{barrier} / k_{\mathrm{B}} T / (1-1/\gamma))$}{%
+    This option sets the $\epsilon$ in Eq.(\ref{eq:opes_potential}).}
+\item %
+  \labelkey{opes_metad|kernel_cutoff}
+  \keydef
+    {kernelCutoff}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    Truncate kernels at this distance, in units of $\sigma$}{%
+    positive decimal}{%
+    $\sqrt{2*\mathrm{barrier}/k_{\mathrm{B}} T/(1-1/\gamma)}$}{%
+    If the value of $(\boldsymbol{\xi}-\boldsymbol{\xi}')^T \Sigma^{-1}(\boldsymbol{\xi}-\boldsymbol{\xi}')$ is larger than the square of this value, then $G(\boldsymbol{\xi}, \boldsymbol{\xi}')$ is evaluated as zero.}
+\item %
+  \labelkey{opes_metad|compression_threshold}
+  \keydef
+    {compressionThreshold}{%
+    \texttt{opes{\textunderscore}metad}}{%
+    Merge kernels if closer than this threshold, in units of $\sigma$}{%
+    positive decimal}{%
+    1.0}{%
```

Comment:
This keyword is listed as mandatory/compulsory in PLUMED, so maybe it shouldn't have a default value?

Reply:
No. Compulsory keywords in PLUMED could have default values (see https://www.plumed.org/doc-master/user-doc/html/_o_p_e_s__m_e_t_a_d__e_x_p_l_o_r_e.html):
```
COMPRESSION_THRESHOLD 	( default=1 ) merge kernels if closer than this threshold, in units of sigma 
```

Reply:
Thanks!